packages/storefront-lwc/src/modules/commerce/checkout/__tests__/checkout.test.js

```diff
@@ -0,0 +1,193 @@
+import { createElement } from 'lwc';
+import { mockQuery, reset } from '@lwce/apollo-client';
+import Checkout from 'commerce/checkout';
+
+import { registerWireService } from '@lwc/wire-service';
+import { register } from 'lwc';
+
+registerWireService(register);
+
+function tick(time = null) {
+    return new Promise(resolve => setTimeout(resolve, time));
+}
+
+describe('<commerce-checkout>', function() {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+
+        reset();
+    });
+
+    it('should render a loading page', async function() {
+        const element = createElement('commerce-checkout', {
+            is: Checkout,
+        });
+
+        document.body.appendChild(element);
+
+        await tick();
+
+        expect(element.shadowRoot.querySelector('.loading')).toMatchSnapshot();
+    });
+
+    it('should initially show the shipping stage', async function() {
```

Comment:
Just two simple tests to start. As the subcomponents are built out, naturally we'll add more tests.